tools/go-agent/instrument/plugins/rewrite/var.go

```diff
@@ -19,15 +19,14 @@ package rewrite
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/dave/dst"
 )
 
 func (c *Context) Var(val *dst.ValueSpec, onlyName bool) {
 	if len(val.Names) == 1 {
 		oldName := val.Names[0].Name
-		if !strings.HasPrefix(oldName, GenerateCommonPrefix) {
+		if !c.alreadyGenerated(oldName) {
```

Comment:
Something could be rewritten twice? Could you explain the case?

Reply:
Yes, I could rewrite it twice. 
1. Generate name mapping relationships for all top-level types, which will be used by all Go files that reference them to quickly find mappings based on names. At this point, it is used only for name changes.
2. Once the top-level types are generated, all content will be rewritten. This method may be executed again, and will also be re-executed when variables are defined within the method.